### PR TITLE
Intial Commit

### PR DESCRIPTION
**Pull request by breakcodes**
This is a web app for doctor search engine.

### CODE REVIEW

File: controllers/routes.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+var appDir = path.dirname(require.main.filename)
+
+router.get('/', function(req, res) {
+  res.sendFile(appDir+'/views/index.html')
+})
+
+//Api to fetch profile of a particular file
+router.get('/profile/:id',function(req,res) {
+    apiUrl = 'https://api.practo.com/doctors/'+req.params.id+'?with_relations=true'
```
Comment (shreedharshetty) [inline controllers/routes.js]: should never forget vars

File: controllers/routes.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    {
+        url : apiUrl,
+        headers : 
+        {
+            'X-CLIENT-ID' : config.clientId,
+            'X-API-KEY' : config.token
+        }
+    },function(error,response,body){
+        if(!error && response.statusCode == 200) 
+        {
```
Comment (shreedharshetty) [inline controllers/routes.js]: bhai, parenthesis should be placed at the same line 
`if(!error && response.statusCode == 200) {`

File: controllers/routes.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        else
+        {
+            console.log('error While fetching')
+        }
+    })
+})
+
+
+//Api to fetch doctors based on specialization and locality
+router.get('/search0/:locality/:specialization/:offset',function(req,res){
```
Comment (shreedharshetty) [inline controllers/routes.js]: what is search0?

File: index.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+var express = require('express')
+var app = express()
+var router = express.Router();
+var $ = require('jQuery');
```
Comment (shreedharshetty) [inline index.js]: what is jquery doing here?

File: public/css/style.css [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+#navb
+{
+position:relative;
+margin-top:180px;
+}
+
+
+.image-header
```
Comment (shreedharshetty) [inline public/css/style.css]: same for parenthesis .image-header {

File: public/javascript/search.js [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+						'</div>'+
+						 '</a>'+
+						'</div>'+
+						'<div class=\'row\'>'+
+						'<hr class=\'half-line\'>'+
+						'</div>'
+		}
+	//Displaying doctor info on the card
+	$('#card').html(doctorList)	
+	})
```
Comment (shreedharshetty) [inline public/javascript/search.js]: should never miss semicolons
also try integrating eslint for this project

**Comment by breakcodes** [comment]
I have made the changes as suggested by you.

**Pull request merged**